netket/sampler/abstract_sampler.py

```diff
@@ -27,11 +27,11 @@ def reset(self, init_random=False):
         pass
 
     @property
-    def machine_func(self):
-        return lambda x, out=None: _np.square(_np.absolute(x), out)
+    def machine_pow(self):
+        return 2.0
```

Comment:
I don't understand why there is this.

Reply:
Wouldn't it just be easier to have `machine_pow` be a standard property with default value 2.0 in the base abstract class?